### ISSUE TITLE
Support for combined stdout and stderr output

### ISSUE THREAD

**Issue by sipsma** [dagger/dagger#3496]
There's many times you want to get both the stdout and stderr. It's slightly tedious to do this with the Go SDK since you have to grab them both separately (and check err, etc.). Maybe there could just be a field `Output` that grabs both for you at the same time?

**Comment by sipsma** [comment]
To be clear, the first part of this issue is deciding if we actually want it. If so, the implementation should be straightforward I think.

**Comment by helderco** [comment]
Yes I think this makes a lot of sense. Problem is, you still need to specify the fields, no?

```graphql
container {
  from(...) {
    exec(...) {
        output {
            stdout
            stderr
        }
    }
  }
}
```

So it still makes two queries:
```go
cmd := ....Exec().Output()
stdout, err := cmd.Stdout(ctx)
stderr, err := cmd.Stderr(ctx)
```

What can make this better is being able to select sibling fields in the same query since you'd execute only once.

**Comment by vito** [comment]
@helderco The problem with fetching separate fields is you lose the interleaving, i.e. what a human would actually see if they ran the command themselves. I think that's the root ask of this issue.

Implementation wise this might be a little tricky since we currently direct stdout/stderr to separate files. We could have the shim duplex to both separate streams _and_ a combined one, it'll just add a bit of overhead. Probably not a huge deal.

**Comment by helderco** [comment]
I see, so you mean having 3 fields:

```graphql
container {
  from(...) {
    exec(...) {
        stdout
        stderr
        output  # string with combined stdout + stderr
    }
  }
}
```
